ansible/files/telemetry.sh

```diff
@@ -122,34 +186,35 @@ wget -nc https://mran.microsoft.com/install/RRO-3.2.1-el6.x86_64.tar.gz
 tar -xzf RRO-3.2.1-el6.x86_64.tar.gz
 rm RRO-3.2.1-el6.x86_64.tar.gz
 cd RRO-3.2.1; sudo ./install.sh; cd ..
-$ANACONDAPATH/bin/pip install rpy2
-mkdir -p $HOME/R_libs
+$ANACONDA_PATH/bin/pip install rpy2
+
+# Setup global bash file
+# .sh files in /etc/profile.d/ run whenever a user logs in
+GLOBAL_BASHRC=/etc/profile.d/atmo.sh
```

Comment:
Hmm. I suppose we could, but we use a whole lot of options in this file ($ANACONDAPATH, memory configs, etc.) which means changes here would need to be paralleled in that new file. I think it makes sense to keep it contained here for now.